Demo/Cartfile

```diff
@@ -0,0 +1,2 @@
+github "https://github.com/braintree/braintree_ios.git" "remove-spmtest"
```

Comment:
Currently, we're using a branch where we deleted the SPMTest demo app (since it causes Carthage to timeout if it's present). Once we do another release of the base SDK, we'll update the Cartfile to use that release. It'll pull the pre-built frameworks from the GitHub release instead of building them, so we won't have an issue with the SPMTest app.